src/main/osd/osd_elements.c

```diff
@@ -577,8 +582,8 @@ static void osdElementAntiGravity(osdElementParms_t *element)
 }
 
 #ifdef USE_ACC
-static void osdElementArtificialHorizon(osdElementParms_t *element)
-{
+
+static void osdElementArtificialHorizon(osdElementParms_t *element){
```

Comment:
Coding style in Betaflight is 1TBS - opening brackets go onto the next line for function definitions.